.github/actions/lint-front-matter/rules/tags-are-valid.js

```diff
@@ -0,0 +1,35 @@
+const id = 'tags-are-valid';
+const Rule = require('./rule');
+const {ok, fail, warn} = require('../utils/status');
+const postTags = require('../../../../src/site/_data/postTags');
```

Comment:
I wonder if this should be renamed `possibleTags` or something like that. This name caused me to think the inclusion logic at line 18 was inverted. In other words, `postTags` sounds more like a name for tags in a single post than it does for the list of all tags.

Reply:
mmmm maybe. But we should do that in a separate PR. There's a lot of things depending on `postTags` and we need to update all those files.